org.osgi.test.junit4/src/main/java/org/osgi/test/common/annotation/PropertiesConverter.java

```diff
@@ -17,173 +17,19 @@
  *******************************************************************************/
 package org.osgi.test.common.annotation;
 
-import java.lang.reflect.Array;
-import java.util.Arrays;
 import java.util.Dictionary;
-import java.util.Hashtable;
-
-import org.osgi.test.common.annotation.Property.Scalar;
-import org.osgi.test.common.annotation.Property.Type;
 
+/**
+ * The generic PropertiesConverter is unable to support all forms of property
+ * conversion and should not be used. Use the JUnit version specific alternative
```

Comment:
Since this is in the junit4 project's source folder, why not have the comment refer to the specific `org.osgi.test.junit4.properties.PropertiesConverter` replacement class?